templates/c2s-gp-finder/config.yml

```diff
@@ -0,0 +1,6 @@
+name: C2S - GP Finder
+description: |
+  Connecting to Services - GP Finder (require separate loadbalancer stack or traefik lb running in the stack)
+version: v1
+category: nhsuk beta
+projectURL: https://github.com/nhsuk/nhsuk-rancher-templates
```

Comment:
Hm, that might be the reason the lb/c2s0 template was out of date as the config was pointing at the same url?